src/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<!--[if lt IE 7 ]> <html lang="en" class="ie6"> <![endif]-->
+<!--[if IE 7 ]>    <html lang="en" class="ie7"> <![endif]-->
+<!--[if IE 8 ]>    <html lang="en" class="ie8"> <![endif]-->
+<!--[if IE 9 ]>    <html lang="en" class="ie9"> <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en"> <!--<![endif]-->
+<html lang="en" class="no-js"></html>
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="apple-touch-icon" href="icon.png">
+  <title>carbon</title>
+  <script>
+    window.__PROMISE_POLYFILL__ = "${webpack.publicPath}" + <% print(JSON.stringify(
+        webpack.chunks.find(c => c.names.includes('promisePolyfill')).files[0]
+    )) %>;
+  </script>
+</head>
+<body>
+  <h1>carbon</h1>
+</body>
+</html>
```

Comment:
Is this a demo site? If this has a purpose, we should probably describe it in the README.

Reply:
I definitely need to update the readme. I don't know whether it makes sense to have some source files in here, since we discussed focusing primarily on configs for now. But since webpack is such a major thing to configure, I ended up adding a more complete demo build, thinking that a more complete context would be helpful. Is it too much?

Reply:
I like having a more complete context. I think that gives it more flexibility in the way that it gets used. People can either use Carbon as a jumping off point for their own project or mine it for configs.